benchmark/atoi-corpus.h

```diff
@@ -1,9 +1,9 @@
 #include "atoi.h"
-
 #include <vector>
 #include <string>
 #include <cstring>
 #include <random>
+#include <zoo/pp/platform.h>
```

Comment:
Micro-nit:

In this codebase, please always #include from the most specific to the more general, like this:
1. Headers specific to the feature related to the current file
2. General zoo headers (`platform.h` should probably be the last)
3. Specific external librareis
4. The standard library.

In this way, you give the chance to the local code to provide overrides for more general code.

Reply:
that makes sense !